app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_type/AddBusStopType.java

```diff
@@ -0,0 +1,52 @@
+package de.westnordost.streetcomplete.quests.bus_stop_type;
+
+import android.os.Bundle;
+
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.OverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBusStopType extends OverpassQuestType
+{
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes with public_transport=platform and !shelter and !informal";
```

Comment:
I'd support the "old" tagging scheme `highway=bus_stop` as well, as it is still in widespread use: about 1.5M elements for bus_stop and about half of it for `public_transport=platform`. No reason to miss out on improving them, right?

> Informal

This doesn't seem to be documented. I am reluctant to introduce own tags with this app that haven't been agreed on in the community. https://wiki.openstreetmap.org/wiki/Tag:public_transport%3Dplatform
Where do you have the information from that informal can/should be used for these cases?
Also, while the child on the picture is very cute, it is problematic to ask for this in this app because:
- it takes someone to map an "informal" stop in the first place
- if that person did not tag it as informal already, for anyone else passing by that place, the bus stop mapped here seems like an error. In the best case, he will answer with a note "there is no bus stop here", in the worst case, he will just delete it. Keyword "mapping what is on the ground" - well, there is nothing on the ground. Noone can confirm if it is an error here or if it is "informal".
- informal bus stops will be in the vast minority, exceptional answers should not clutter the main interface and can even lead to them being wrongfully chosen

So all in all, I think it makes more sense to simply ask if this bus stop is sheltered or not (simple yes/no) and don't ask if "informal" tag is present. What do you think?